### ISSUE TITLE
about the dataset partition

### ISSUE THREAD

**Issue by zsrluminous** [salmank255/smoke-detection#2]
I read your paper , and am interested in the content of your paper. Recently, I want to realize the idea of your article. I have some questions about the dataset partition. Please help me to answer them when you are free. 

I see the paper said " the dataset comprises of a total of 72012 images, extracted from videos. For the evaluation of our system, we used our recent strategy [11] by using 20% data for training and the rest of 30% and 50% for validation and testing, respectively." in page 9 .

Because the images data come from the video, the similarity between them in a same video is very high. I would like to ask if there is any cross between the images in training set, validation set and test set in this article? Will different frames from the same video appear in the training, validation or test set respectively.   Or, the videos in the training set will not reappear in the validation set and the test set. 
Thank you very much.

**Comment by hali1122** [comment]
I have the same questions about the dataset partition
